go/chat/inboxsource.go

```diff
@@ -584,7 +585,7 @@ func (s *HybridInboxSource) Read(ctx context.Context, uid gregor1.UID,
 	localizerTyp types.ConversationLocalizerTyp, useLocalData bool, maxLocalize *int,
 	query *chat1.GetInboxLocalQuery, p *chat1.Pagination) (inbox types.Inbox, localizeCb chan types.AsyncInboxResult, err error) {
 
-	defer s.Trace(ctx, func() error { return err }, "Read")()
+	defer s.Trace(ctx, func() error { return err }, fmt.Sprintf("Read: query: %v", query))()
```

Comment:
Topic names could be in here, we cant log this.

Reply:
Also I'm not a huge fan of spamming out too much logging here, I'd prefer to keep this log line terse.

Reply:
ah gotcha